core/src/main/resources/defaults/clustertemplates/gcp/gcp-streaming-723.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "7.2.3 - Streams Messaging Heavy Duty for Google Cloud",
+  "description": "",
+  "type": "STREAMING",
+  "cloudPlatform": "GCP",
+  "distroXTemplate": {
+    "cluster": {
+      "blueprintName": "7.2.3 - Streams Messaging Heavy Duty: Apache Kafka, Schema Registry, Streams Messaging Manager"
+    },
+    "externalDatabase": {
+      "availabilityType": "HA"
```

Comment:
Nice, did we implement an external database for the datahub already?

Reply:
removed it